src/Sulu/Bundle/MarkupBundle/Resources/config/services.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="sulu_markup.tag.registry" class="Sulu\Bundle\MarkupBundle\Tag\TagRegistry">
+            <argument type="collection"/>
+        </service>
+
+        <service id="sulu_markup.parser" class="Sulu\Bundle\MarkupBundle\Markup\HtmlMarkupParser">
+            <argument type="service" id="sulu_markup.tag.registry"/>
+
+            <tag name="sulu_markup.parser" type="html"/>
+        </service>
+
+        <service id="sulu_markup.response_listener" class="Sulu\Bundle\MarkupBundle\Listener\MarkupListener">
+            <argument type="collection"/>
+            <argument type="collection">
+                <argument type="string" key="text/html">html</argument>
+                <argument type="string" key="text/xml">xml</argument>
+                <argument type="string" key="application/xml">xml</argument>
```

Comment:
I would move that somehow to the `Configuration` and `Extension` class. Or can we maybe even use [Symfony builtins](http://symfony.com/doc/current/cookbook/request/mime_type.html) instead?